app/src/containers/CartPage.tsx

```diff
@@ -179,30 +247,72 @@ class CartPage extends React.Component<RouteComponentProps, CartPageState> {
     return ('');
   }
 
+  handleModalOpen() {
+    this.setState({
+      openModal: true,
+    });
+  }
+
+  handleModalClose() {
+    this.setState({
+      openModal: false,
+    });
+  }
+
+  renderCartTabs() {
+    const { cartsData } = this.state;
+    return (
+      <div className="cart-tabs-container">
```

Comment:
For this element, we need to handle the cases (as in the mockup) where we have more carts than will fit in the tab bar. A menu on the right should show the remaining carts:
![image](https://user-images.githubusercontent.com/39800563/66667754-c08c3200-ec21-11e9-9933-cb987e7317ca.png)

Reply:
**Update:** this is expected to change. I'll post context over slack.